### PR TITLE
Mistral Large 3 NVFP4 TRTLLM MoE support

### PR DESCRIPTION
**Pull request by elvischenv**
## Motivation

Support Mistral Large 3 NVFP4 TRTLLM MoE.
Tested with cmd from #14485 + `--moe-runner-backend flashinfer_trtllm`:

### Accuracy
TRTLLM MoE:
```
|Tasks|Version|     Filter     |n-shot|  Metric   |   |Value |   |Stderr|
|-----|------:|----------------|-----:|-----------|---|-----:|---|-----:|
|gsm8k|      3|flexible-extract|     8|exact_match|↑  |0.9249|±  |0.0073|
|     |       |strict-match    |     8|exact_match|↑  |0.7180|±  |0.0124|
```
Default cutlass MoE:
```
|Tasks|Version|     Filter     |n-shot|  Metric   |   |Value |   |Stderr|
|-----|------:|----------------|-----:|-----------|---|-----:|---|-----:|
|gsm8k|      3|flexible-extract|     8|exact_match|↑  |0.9272|±  |0.0072|
|     |       |strict-match    |     8|exact_match|↑  |0.7104|±  |0.0125|
```

### Perf
TRTLLM MoE:
```
Output token throughput (tok/s):         15138.92
Median TPOT (ms):                        58.37
```
Default cutlass MoE:
```
Output token throughput (tok/s):         8677.97
Median TPOT (ms):                        80.68
```

## Modifications

Support Compressed Tensors W4A4 NVFP4 TRTLLM MoE method.

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [ ] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).
- [ ] Work with maintainers to merge your PR. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process)

### CODE REVIEW

File: python/sglang/srt/layers/quantization/utils.py [hunk 1]
```diff
@@ -592,3 +592,145 @@ def swizzle_blockscale(scale: torch.Tensor):
...
+    s1, s3 = scale.split(half, dim=dim)
+
+    return (
+        torch.cat([w3, w1], dim=dim).contiguous(),
+        torch.cat([s3, s1], dim=dim).contiguous(),
+    )
+
+
+def prepare_static_weights_for_trtllm_fp4_moe(
+    # args_dequant,
```
Comment (ispobock) [inline python/sglang/srt/layers/quantization/utils.py]: Maybe we can remove these two additional comments.

**Review by ispobock — APPROVED** [review]
LGTM

File: python/sglang/srt/layers/quantization/utils.py [hunk 2]
```diff
@@ -592,3 +592,145 @@ def swizzle_blockscale(scale: torch.Tensor):
...
+    # args,
```
Comment (elvischenv) [inline python/sglang/srt/layers/quantization/utils.py]:
```suggestion
```
